lib/screenplay_web/router.ex

```diff
@@ -94,7 +94,7 @@ defmodule ScreenplayWeb.Router do
     get("/active_alerts", AlertController, :active_alerts)
     get("/past_alerts", AlertController, :past_alerts)
 
-    get("/station_screen_orientation_list", PageController, :station_screen_orientation_list)
+    get("/outfront_takeover_tool_screens", PageController, :outfront_takeover_tool_screens)
```

Comment:
It looks like this is already in a `ScreenplayWeb.OutfrontTakeoverTool` scope, so is it redundant to specify `outfront_takeover_tool` in the path / action name? (though confusingly it shares an `/api` path prefix with other, non-OFTT routes...)